in_progress/roles.md

```diff
@@ -0,0 +1,89 @@
+# Abstract
+
+The purpose of this document is to formalize the governance process used by the `scverse` project, to clarify how decisions are made and how the various elements of our community interact.
+
+As a whole, we are a consensus-based community project.
+However, any subproject within the scverse can have their own governance system as long as they do not conflict with the rules set out here.
+Anyone with an interest in the project can join the community, contribute to the project design, and participate in the decision making process.
+This document describes how that participation takes place, how to find consensus, and how deadlocks are resolved.
+
+# Roles And Responsibilities
+
+## The Community
+
+The scverse community consists of anyone using or working with the project in any way.
+
+## Contributors
+
+A community member can become a contributor by interacting directly with the project in concrete ways, such as:
+
+- proposing, discussing, or reviewing a change to the code, documentation, or specification via a GitHub pull request to any https://github.com/scverse repository;
+- reporting a GitHub issue to any https://github.com/scverse repository;
+- discussing examples or usage issues on any of our discussion platforms
+
+among other possibilities.
+Any community member can become a contributor, and all are encouraged to do so.
+By contributing to the project, community members can directly help to shape its future.
+
+Potential contributors are encouraged to read the Contributing Guide (**TODO**) as well as the Code of Conduct (**TODO**)
+
+A community member becomes a contributor when the following criteria are met:
+
+- At least two core developers from separate employers agree membership is a good idea.
+- The new member has supported the project several times, either through code or otherwise, and has demonstrated a willingness to help solve other people's problems, not just their own.
+- The new member demonstrates some degree of familiarity of open source practices and online courtesy.
+
+## Core developers
+
+Core developers are community members that have demonstrated continued commitment to the project through ongoing contributions.
```

Comment:
In practice probably. Two cases where this might not make sense:

* Collaboratorive projects that just need an organization to live in. For example, if we are collaborating on something with bioconductor, does that person want the responsibilities of being a "core" member here.
* If we're taking over a project that's otherwise abandoned. On the other hand, why would we do this.

How about: "all primary developers of core projects are invited to be core developers"? Could also leave it vague until we've gone through this a bit.

Reply:
Fine with that.

"If we're taking over a project that's otherwise abandoned. On the other hand, why would we do this."
Yeah, of course. No point in including inactive people.

Reply:
Hmm, thinking through this more and trying to fit it in with:

> New core developers can be nominated by any existing core developer, and for details on that process see our core developer guide.

How about:

> All core packages must have at least one core developer as a maintainer.
> Primary maintainers of new core packages will be invited to join as a core member.

I think there are benefits to requiring a "core maintainer" be listed on a package. That way we won't end up with the `pydata/patsy` situation where a our name is on it, but no one is actually in charge of maintaining it.